src/pages/index.tsx

```diff
@@ -46,7 +48,9 @@ function renderCardList(tasks: task[], isEditable: boolean) {
       content={item}
       key={item.id}
       shouldEdit={isEditable}
-      onContentChange={async (id: string, newDetails: any) => updateCardContent(id, newDetails)}
+      onContentChange={async (id: string, newDetails: any) => {
+        updateCardContent(id, newDetails, isEditable);
```

Comment:
What is your opinion on `isEditable && updateCardContent(params)` , would this be better than passing it as params and checking the condition?

Reply:
Yes, this way will be better than the current one.
I will do the changes

Reply:
@harshith-venkatesh Sorry the branch got deleted. I have raised a new PR #221